common/Strings/en-us/Resources.resw

```diff
@@ -205,33 +205,49 @@
     <value>Uptime:</value>
     <comment>Title for the uptime property of a compute system. (The amount of time its been running)</comment>
   </data>
-  <data name="HyperVAdminAddUser" xml:space="preserve">
-    <value>Add user to {0} administrators group</value>
-    <comment>Locked="{0}" Text for the button to add user to {0} administrators group</comment>
+  <data name="HyperVAdminAddUserButton" xml:space="preserve">
+    <value>Add user to group</value>
+    <comment>Locked="{Hyper-V}" Text for the button to add user to the Hyper-V administrators group</comment>

```

Comment:
Wrong comment?  `value` does not have `Hyper-V`

Reply:
will update thanks